tests/resources/application_config.php

```diff
@@ -1,13 +1,15 @@
 <?php
 
-return [
-    'modules' => [
-        'Go\ZF2\GoAopModule',
+$modules = [
+    'Go\ZF2\GoAopModule',
+];
 
-        // Required to initialize application
-        'Zend\Router',
-    ],
+if (class_exists('Zend\Router\Module')) {
```

Comment:
I think this is fine as we are running inside our tests scope there `Zend\Router\Module` should always be defined.
So no need to check, or am I missing something?

Reply:
When you install zend mvc 2.x there is no explicit zend router module. With zend mvc 3.x there is an explicit zend router module which must be added to your list of modules.

This makes it quite difficult to support both versions with the same tests.

Reply:
Didn't check that one.. fine with me then :)